specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/preview/2019-02-01-preview/imagebuilder.json

```diff
@@ -603,6 +603,13 @@
         "script": {
           "type": "string",
           "description": "The shell script to be run for customizing. It can be a github link, SAS URI for Azure Storage, etc"
+        },
+        "inline": {
```

Comment:
nit: a better command would be inlineCommands or commands. Not blocking on this.